mars/deploy/oscar/rayconfig.yml

```diff
@@ -11,7 +11,9 @@ cluster:
   node_timeout: 120
   node_check_interval: 1
   ray:
-    supervisor_exclusive_node: no
+    supervisor:
+      standalone: no
```

Comment:
```suggestion
      standalone: false
```

Reply:
Why change the default standalone value to false?

Reply:
Is true/false a better literal name in yaml?

Reply:
YAML supports `true / false` and `yes / no`. I followed the mars's `config.yml`, choose `yes / no` instead of `true / false`. e.g. https://github.com/mars-project/mars/blob/master/mars/deploy/oscar/config.yml